### ISSUE TITLE
detached dialog windows in QT loose focus making it impossible to use them

### ISSUE THREAD

**Issue by aspizu** [phillbush/shod#25]

**Comment by phillbush** [comment]
How does it lose focus? When clicking on the dialog window it gain focus.  
Are you referring to losing focus when focusing another window and then focusing back the original one?

**Comment by aspizu** [comment]
no, clicking on the window just focuses the window behind it and it sometimes blinks

**Comment by phillbush** [comment]
I could not reproduce this error on a qt application I use (qbittorrent). What application are you using that gets this issue?

**Comment by aspizu** [comment]
Virtual Box and using kvantum

**Comment by phillbush** [comment]
Could not reproduce this issue, sorry.

I tried VirtualBox with kvantum on openSUSE and still could focus the detached dialog without problem.

Does that only works on qt/kvantum applications?

**Comment by aspizu** [comment]
did you try pressing buttons?
